### PR TITLE
Docs skeleton

### PR DESCRIPTION
**Pull request by musnit**
Basic skeleton for docs done - also wrote out components

### CODE REVIEW

**Comment by vgeddes** [comment]
Was curious why you added a redundant `markdown` directory? It adds an extra level of nesting which seems unnecessary.

**Comment by musnit** [comment]
> Was curious why you added a redundant `markdown` directory? It adds an extra level of nesting which seems unnecessary.

lies, its cleaner :P

File: docs/markdown/concepts-and-architecture/components.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+```
+
+Payloads for an EthereumRPC are expected to be ABI encoded contract call data which have been encoded by the sending pallet before sending. The receiving smart contract will be called directly with that ABI-encoded call.
+
+#### Snowbridge Parachain
+With our bridge running on Polkadot, we aren't just verifying and sending messages on the Polkadot relay chain. We also want to be able to send and verify messages on any parachain, and support assets and functionality that are not supported on the relay chain. Snowbridge runs its own parachain to facilitate this.
+
+For more details, see [Snowbridge Parachain](./snowbridge-parachain)
+
+#### <a name="parachain-message-commitment"></a> Parachain Message Commitment
```
Comment (vgeddes) [inline docs/markdown/concepts-and-architecture/components.md:91]: Shouldn't be necessary to add these anchors manually. The Markdown processor will generate an anchor automatically by downcasing the header text.

So can use something like this to link to this section:

```markdown
blah blah [foo](./components#parachain-message-commitment) blah blah
```

The anchor `#parachain-message-commitment` should be generated automatically by the Markdown processor by downcasing the

File: docs/markdown/concepts-and-architecture/components.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+Our parachain facilitates any parachain that wants to send messages out to Ethereum, and so is responsible for storing those messages in a way that can be verified trustlessly on Ethereum. It does so through creating custom message commitments on our parachain that can be easily and cheaply verified.
+
+*These parachain commitments are effectively the sending part of a channel for communication from Polkadot to Ethereum. We may rename/refactor this functionality for consistency in future*
```
Comment (vgeddes) [inline docs/markdown/concepts-and-architecture/components.md:94]: rename/refactor => modify

File: docs/markdown/concepts-and-architecture/components.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+For trustless verification on Ethereum we need a Polkadot light client and a parachain light client for our parachain. We have a set of smart contracts that run on Ethereum to provide this.
+
+For more details, see [Polkadot Light Client Verifier](./polkadot-light-client-verifier/)
+
+## Basic Bootstrap Bridge
+The basic bootstrap bridge is a lower level bridge intended to act as a bootstrap and simpler alternative to the main bridge. It is not incentivized and so does not guarantee delivery. It does however guarantee deliverability and replay protection.
+
+This bridge consists of 2 basic channels.
+
+### Basic Ethereum to Polkadot Message Channel
```
Comment (vgeddes) [inline docs/markdown/concepts-and-architecture/components.md:108]: Too Wordy.

Rather: Basic Message Channel (Ethereum → Polkadot)

File: docs/markdown/concepts-and-architecture/components.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+The Basic Polkadot to Ethereum Message Channel is a channel for sending Ethereum RPCs out from the parachain to Ethereum. It is responsible for accepting requests from other pallets and parachains for messages to be sent over to Ethereum. It puts those requests into [Parachain Message Commitments](#parachain-message-commitment) that will be included in the parachain header. The channel then processes those commitments and verifies them via the [Polkadot and Parachain Light Client Verifier](#polkadot-light-client-verifier) to extract Ethereum RPCs. Those Ethereum RPCs are then routed to their target contract by calling that contract.
+
+For more details, see [Basic Polkadot to Ethereum Message Channel](./basic-polkadot-to-ethereum-message-channel)
+
+## Incentivized Bridge
+The incentivized bridge extends the basic bootstrap bridge by adding a strict message ordering channels in both directions that enforce delivery of messages in the order that they are sent. It also adds incentives which provide guaranteed delivery with strong properties based on simple economic assumptions without the need for any kind of Oracle-based system.
```
Comment (vgeddes) [inline docs/markdown/concepts-and-architecture/components.md:121]: Can leave out the word "bootstrap" for less wordiness.

Also "Oracle-based solution" instead of "system"

File: docs/markdown/concepts-and-architecture/ethereum-light-client-verifier.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+layout: default
+title: Ethereum Light Client Verifier
+nav_order: 2
+permalink: /concepts/ethereum-light-client-verifier
+parent: Concepts and Architecture
+---
+# Ethereum Light Client Verifier
+This is a Rust pallet that is responsible for watching and verifying Ethereum Mainnet. It runs a light client that can track Ethereum as it progresses. It accepts new Ethereum headers once they are produced, checks proof of work and follows the longest chain. It also handles short-term forks up to some number of confirmations. It is also responsible for verifying actual messages that are coming across the bridge from Ethereum smart contract applications.
+
+## Proof of Work Verification
```
Comment (vgeddes) [inline docs/markdown/concepts-and-architecture/ethereum-light-client-verifier.md:11]: Proof-of-Work Verification

File: docs/markdown/concepts-and-architecture/ethereum-light-client-verifier.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+The proof of work verification is done based on Ethash verification as described in https://eth.wiki/en/concepts/ethash/ethash. One caveat is that typical light clients will store the full cache used to generate the hashing dataset. On our parachain, even the cache is too big (~64MB+ at the moment), so instead of storing it directly, we store a merkle root that commits to 2 - 4 years worth of cache data. As part of the proof of work verification, the relayer must submit a merkle proof of the data used for generating the proof of work on each block.
```
Comment (vgeddes) [inline docs/markdown/concepts-and-architecture/ethereum-light-client-verifier.md:12]: Rather:

```
The proof-of-work (PoW) verification is implemented using an [Ethash](https://eth.wiki/en/concepts/ethash/ethash) verifier.
```

Comment (vgeddes) [reply docs/markdown/concepts-and-architecture/ethereum-light-client-verifier.md:12]: Rather:

2 to 4 years worth

File: docs/markdown/concepts-and-architecture/ethereum-light-client-verifier.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+
+This commitment will need to be updated every 2+ years, and is expected to be done so via decentralized governance which will be implemented post-launch.
+
+## Transaction Receipt proofs
```
Comment (vgeddes) [inline docs/markdown/concepts-and-architecture/ethereum-light-client-verifier.md:16]: _proofs_ should be title-case

File: docs/markdown/concepts-and-architecture/ethereum-light-client-verifier.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+The verifier needs to also verify messages from Ethereum smart contracts. To do so, we verify Ethereum Transaction Receipt proofs which contain the event logs of all events in the Ethereum transaction. Transaction Receipts also include additional data, for example transaction input data, but we primarily use the event data as our medium for bridge messages and so only extract event data for now.
```
Comment (vgeddes) [inline docs/markdown/concepts-and-architecture/ethereum-light-client-verifier.md:17]: >  To do so, we verify Ethereum Transaction Receipt proofs which

Rather: To do so, we verify proofs for transaction receipts, which ...

File: docs/markdown/using-snowchain/building-with-snowbridge/running-locally.md [hunk 9]
```diff
@@ -0,0 +1,59 @@
+---
+layout: default
+title: Running and Testing the Bridge Locally
+parent: Building With Snowbridge
+permalink: /building-with-snowbridge/running-locally
+nav_order: 1
+---
+
+# Running the bridge locally and testing it end to end manually from UI yourself
```
Comment (vgeddes) [inline docs/markdown/using-snowchain/building-with-snowbridge/running-locally.md:9]: This document is _really_ out of date (if its from the same PR submitted months ago) and will only lead to confusion and frustration for whoever tries to follow it.

I suggest we remove it.

File: docs/markdown/using-snowchain/building-with-snowbridge/xcm-for-state.md [hunk 10]
```diff
@@ -0,0 +1,46 @@
+---
+layout: default
+title: XCM Interface for Arbitrary State and Cross Chain dApps
+parent: Building With Snowbridge
+permalink: /building-with-snowbridge/xcm-for-state/
+nav_order: 3
+---
+
+# XCM Interface for Arbitrary State and Cross Chain dApps
```
Comment (vgeddes) [inline docs/markdown/using-snowchain/building-with-snowbridge/xcm-for-state.md:9]: Too wordy. Rather:

XCM Interface for building cross-parachain applications

File: docs/markdown/using-snowchain/building-with-snowbridge/xcm-for-state.md [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+
+XCMP messages are asynchronous and sent in a fire-and-forget manner. Messages sent from Parachains to Ethereum will need to specify a target contract for delivery. Parachains will need to register to be notified of messages coming from Ethereum, and will be notified when relevant messages come through for them.
+
+At the application layer, parachain apps and ethereum smart contracts that interact are responsible for being aware of and trusting each other and for determining the payload and interface of their own messages. The bridge just facilitates transfer, verification and routing of these messages to the requested target application. Our ETHApp and ERC20App are examples of pairs of custom FRAME + solidity applications that trust each other and specify a shared interface - although they're implemented as pallets, one could imagine them working similarily as seperate parachains.
+
+Example: Register a custom FRAME application to receive messages
+```
+Xcm::Transact {
```
Comment (vgeddes) [inline docs/markdown/using-snowchain/building-with-snowbridge/xcm-for-state.md:24]: This is obsolete and doesn't reflect our IRL discussions on Tues. I'll update it myself. 😄

File: docs/markdown/index.md [hunk 12]
```diff
@@ -0,0 +1,26 @@
...
+nav_order: 1
+permalink: /
+---
+
+# Snowbridge Documentation
+{: .fs-9 }
+
+Welcome to the documentation site for Snowbridge, a trustless bridge system for bridging arbitrary data between Polkadot, Parachains and Ethereum.
+
+[Snowbridge Website](https://snowbridge.snowfork.com/){: target="_blank" .btn-purple .btn .fs-5 .mb-4 .mb-md-0 }
```
Comment (vgeddes) [inline docs/markdown/index.md:13]: These buttons should all be on 1-line for better visual appeal. Maybe remove the newlines between them to achieve that.

File: docs/markdown/playing-with-snowbridge.md [hunk 13]
```diff
@@ -0,0 +1,8 @@
+---
+layout: default
+title: Playing with Snowbridge
+nav_order: 4
+permalink: /playing-with-snowbridge
+---
+# Playing with Snowbridge
```
Comment (vgeddes) [inline docs/markdown/playing-with-snowbridge.md:7]: Need to add a in-progress chip element here:

```
In-Progress
{: .label .label-yellow }
```

File: docs/markdown/using-snowchain/validating-and-relaying/running-a-collator-node.md [hunk 14]
```diff
@@ -0,0 +1,8 @@
+---
+layout: default
+title: Running a Collator Node
+parent: Validating and Relaying
+nav_order: 1
+---
+# Running a Collator Node
```
Comment (vgeddes) [inline docs/markdown/using-snowchain/validating-and-relaying/running-a-collator-node.md:7]: Add:
```
In-Progress
{: .label .label-yellow }
```

File: docs/markdown/validating-and-relaying.md [hunk 15]
```diff
@@ -0,0 +1,9 @@
+---
+layout: default
+title: Validating and Relaying
+nav_order: 5
+has_children: true
+permalink: /validating-and-relaying/
+---
+# Validating and Relaying
```
Comment (vgeddes) [inline docs/markdown/validating-and-relaying.md:8]: Add:

```
In-Progress
{: .label .label-yellow }
```

File: docs/markdown/concepts-and-architecture/polkadot-light-client.md [hunk 16]
```diff
@@ -0,0 +1,50 @@
...
+
+## Proof Creation
+We create various custom proofs that will be used for verification on Ethereum.
+
+### Polkadot Relay Chain Proofs
+To facilitate our bridge, the Polkadot relay chain will have a new gadget, BEEFY, added that will produce merkle proofs signed by Polkadot validators that commit to both new Polkadot Relay chain blocks and new parachain blocks in a form that is cheaply verifiable on Ethereum. Producing these commitments will be mandatory for Polkadot validators, and will have additional new slashing conditions associated with them, and so will ensure that verification of and trust in these custom commitments is as strong as direct verification of Polkadot Consensus.
+
+Following these commitments will effectively allow us to follow both (1) new Polkadot Relay chain block headers and (2) new Parachain block headers.
+
+For more details, see [BEEFY](https://github.com/paritytech/grandpa-bridge-gadget){: target="_blank"}
```
Comment (vgeddes) [inline docs/markdown/concepts-and-architecture/polkadot-light-client.md:27]: Rather let the user decide whether to open the link in a new window (Also I think is could possibly be configured globally using the Jekyll config)

File: docs/markdown/concepts-and-architecture/polkadot-light-client.md [hunk 17]
```diff
@@ -0,0 +1,50 @@
...
+
+## Proof Verification
+Proof verification happens in Solidity and has 3 main steps.
+
+### Following the Polkadot Relay Chain
+The first step for trustless verification of our bridge on Ethereum starts with following the Polkadot relay chain via following new BEEFY MMR proofs *(as mentioned above)* as they are produced and verifying their validity. Their validity is verified by checking that they are signed by the correct set of Polkadot validators.
+
+For more details, see [Polkadot Relay Chain Interactive Update Protocol](/concepts/polkadot-light-client-verifier/interactive-protocol)
+
+### Updating Relay Chain Consensus and Following Snowbridge Parachain
```
Comment (vgeddes) [inline docs/markdown/concepts-and-architecture/polkadot-light-client.md:42]: Too wordy

File: docs/markdown/concepts-and-architecture/polkadot-light-client/parachain-light-client.md [hunk 18]
```diff
@@ -0,0 +1,14 @@
+---
+layout: default
+title: Snowbridge Parachain Light Client
+nav_order: 4
+permalink: /concepts/polkadot-light-client-verifier/parachain-light-client
+parent: Polkadot Light Client Verifier
+grand_parent: Concepts and Architecture
+---
+# Snowbridge Parachain Light Client
```
Comment (vgeddes) [inline docs/markdown/concepts-and-architecture/polkadot-light-client/parachain-light-client.md:9]: Too wordy. Rather:

Light Client for Snowbridge Parachain

File: docs/markdown/concepts-and-architecture.md [hunk 19]
```diff
@@ -0,0 +1,29 @@
...
+## Layered Architecture
+Snowbridge has a layered architecture with a clear seperation between low level bridge functionality, trust functionality and application functionality.
+
+If you're familiar with the conventional [OSI communication model](https://en.wikipedia.org/wiki/OSI_model), our system is a similar simplified version.
+
+You can have communication go from one layer up or down to the next - for example, between the App Layer and the Bridge Layer, as in the green arrow below.
+
+You also have a layer-specific protocol across a single layer, where components on different chains at the same layer communicate to eachother via that protocol - for example, an app-specific protocol between the App Layer on Polkadot and the App Layer on Ethereum, as in the orange arrow below.
+
+![Layered Architecture](/images/layered-architecture.png){: style="max-width: 200%" }
```
Comment (vgeddes) [inline docs/markdown/concepts-and-architecture.md:21]: These diagrams, well, erm, they are certainly unique. I think we can do better than MS-Paint though. 😄 

I'll offer to redesign them using Visio.

File: docs/markdown/concepts-and-architecture.md [hunk 20]
```diff
@@ -0,0 +1,29 @@
...
+permalink: /concepts/
+---
+
+# Concepts and Architecture
+Snowbridge is a complex project with lots of moving pieces that can interact in various configurations. In order to develop a comprehensive understanding of how things work, this section tries to break down and run through those pieces into different concepts and components that should be easier to understand.
+
+## Layered Architecture
+Snowbridge has a layered architecture with a clear seperation between low level bridge functionality, trust functionality and application functionality.
+
+If you're familiar with the conventional [OSI communication model](https://en.wikipedia.org/wiki/OSI_model), our system is a similar simplified version.
```
Comment (vgeddes) [inline docs/markdown/concepts-and-architecture.md:15]: The OSI model actually turned out to be a dead-end and TCP/IP won out in the end. So maybe lets link to that instead?

File: docs/markdown/api-docs.md [hunk 21]
```diff
@@ -0,0 +1,12 @@
...
+layout: default
+title: API Documentation
+nav_order: 7
+permalink: /api-docs
+---
+# API Documentation
+Snowbridge covers multiple codebases including a Rust parachain, various Rust pallets, a Golang relayer, various Solidity smart contracts and frontend code and demos. Our API Documentation across all these codebases is here:
+
+- [Rust pallets and libraries](https://snowbridge-rust-docs.snowfork.com/){: target="_blank"}
+- [Solidity Docs (warning - outdated)](https://github.com/Snowfork/polkadot-ethereum/tree/main/ethereum/docs){: target="_blank"}
```
Comment (vgeddes) [inline docs/markdown/api-docs.md:11]: Rather move the warning outside the anchor text.

File: docs/markdown/api-docs.md [hunk 22]
```diff
@@ -0,0 +1,12 @@
+---
+layout: default
+title: API Documentation
+nav_order: 7
+permalink: /api-docs
+---
+# API Documentation
+Snowbridge covers multiple codebases including a Rust parachain, various Rust pallets, a Golang relayer, various Solidity smart contracts and frontend code and demos. Our API Documentation across all these codebases is here:
+
+- [Rust pallets and libraries](https://snowbridge-rust-docs.snowfork.com/){: target="_blank"}
```
Comment (vgeddes) [inline docs/markdown/api-docs.md:10]: Rather: Rust pallets and crates.

File: docs/markdown/concepts-and-architecture/polkadot-light-client/interactive-protocol.md [hunk 23]
```diff
@@ -0,0 +1,79 @@
+---
+layout: default
+title: Polkadot Relay Chain Interactive Update Protocol
+nav_order: 2
+permalink: /concepts/polkadot-light-client-verifier/interactive-protocol
+parent: Polkadot Light Client Verifier
+grand_parent: Concepts and Architecture
+---
+# Bridge design to a PoW chain based on random sampling
```
Comment (vgeddes) [inline docs/markdown/concepts-and-architecture/polkadot-light-client/interactive-protocol.md:9]: Too Wordy

File: docs/markdown/concepts-and-architecture/polkadot-light-client/interactive-protocol.md [hunk 24]
```diff
@@ -0,0 +1,79 @@
+---
+layout: default
+title: Polkadot Relay Chain Interactive Update Protocol
```
Comment (vgeddes) [inline docs/markdown/concepts-and-architecture/polkadot-light-client/interactive-protocol.md:3]: Too wordy

File: docs/markdown/concepts-and-architecture/polkadot-light-client/mmr-light-client.md [hunk 25]
```diff
@@ -0,0 +1,16 @@
+---
+layout: default
+title: Polkadot Relay Chain MMR Light Client
+nav_order: 3
+permalink: /concepts/polkadot-light-client-verifier/mmr-light-client
+parent: Polkadot Light Client Verifier
+grand_parent: Concepts and Architecture
+---
+# Polkadot Relay Chain MMR Light Client
```
Comment (vgeddes) [inline docs/markdown/concepts-and-architecture/polkadot-light-client/mmr-light-client.md:9]: Too Wordy.

Rather: MMR light-client for Polkadot Relay Chain

File: docs/markdown/concepts-and-architecture/polkadot-light-client/parachain-light-client.md [hunk 26]
```diff
@@ -0,0 +1,14 @@
+---
+layout: default
+title: Snowbridge Parachain Light Client
+nav_order: 4
+permalink: /concepts/polkadot-light-client-verifier/parachain-light-client
+parent: Polkadot Light Client Verifier
+grand_parent: Concepts and Architecture
+---
```
Comment (vgeddes) [inline docs/markdown/concepts-and-architecture/polkadot-light-client/parachain-light-client.md:8]: Needs an in-progress chip:
```
In-Progress
{: .label .label-yellow }
```

File: docs/markdown/core-bridge-applications/polkaerc20.md [hunk 27]
```diff
@@ -0,0 +1,11 @@
+---
+layout: default
+title: PolkaERC20
+parent: Core Bridge Applications
+permalink: /core-bridge-applications/polkaerc20
+nav_order: 2
+---
+# PolkaERC20
+PolkaETH is a cross-chain dApp that facilitates the creation and movement of pegged ERC20 assets in the same way as PolkaETH.
```
Comment (vgeddes) [inline docs/markdown/core-bridge-applications/polkaerc20.md:9]: Typo here ⬆️

File: docs/markdown/core-bridge-applications/polkaeth.md [hunk 28]
```diff
@@ -0,0 +1,11 @@
+---
+layout: default
+title: PolkaETH
+parent: Core Bridge Applications
+permalink: /core-bridge-applications/polkaeth
+nav_order: 1
+---
+# PolkaETH
+PolkaETH is a cross-chain dApp that facilitates the creation and movement of a pegged Ether asset on Polkadot. It allows users to lock up ETH in the PolkaETH application smart contract on Ethereum, and then have that minted as PolkaETH via the PolkaETH pallet on Snowbridge's parachain. PolkaETH can be moved to other parachains and used in their products and applications, or can be burned by users on Snowbridge's parachain to return their value to Ethereum and unlock their Ether out of the PolkaETH smart contract.
```
Comment (vgeddes) [inline docs/markdown/core-bridge-applications/polkaeth.md:9]: dApp sounds too trendy and informal. Rather just "cross-chain application"

File: docs/markdown/concepts-and-architecture/components.md [hunk 29]
```diff
@@ -0,0 +1,125 @@
...
+
+### <a name="polkadot-light-client-verifier"></a> Polkadot and Parachain Light Client Verifier
+For trustless verification on Ethereum we need a Polkadot light client and a parachain light client for our parachain. We have a set of smart contracts that run on Ethereum to provide this.
+
+For more details, see [Polkadot Light Client Verifier](./polkadot-light-client-verifier/)
+
+## Basic Bootstrap Bridge
+The basic bootstrap bridge is a lower level bridge intended to act as a bootstrap and simpler alternative to the main bridge. It is not incentivized and so does not guarantee delivery. It does however guarantee deliverability and replay protection.
+
+This bridge consists of 2 basic channels.
```
Comment (vgeddes) [inline docs/markdown/concepts-and-architecture/components.md:106]: Rather: This bridge consists of two channels.

**Review by vgeddes — COMMENTED** [review]
Looks good! Added a bunch of comments, mostly to do with style and language.

Two concerns to highlight:

1. We shouldn't be referring to messages as RPCs, as the latter term has several meanings, and is mostly used to refer to networked APIs or interface definitions. Anyway, if I recall correctly, when people implement RPC interfaces (like Protobuf, JSON-rpc, etc), the on-wire instructions are usually referred to as messages anyway. 

2. Many of the section titles are overly wordy and dense, to the point where they may not easily make sense for others. For example "Polkadot Relay Chain MMR Light Client" would be easier to understand if it was rewritten as "Light client for the Relay Chain".

File: docs/markdown/concepts-and-architecture/polkadot-light-client/interactive-protocol.md [hunk 30]
```diff
@@ -0,0 +1,79 @@
...
+3. The prover sends the signatures $sig_i$, Merkle proofs of public keys from $r_{val}$ and Merkle proofs of signatures from $r_{sig}$ corresponding to the validators the light client asked for.
+
+4. If all these signatures are also correct then the light client accepts the block.
+
+
+Analysis: If at least $2/3$ of validators are honest but no honest validator signed $S$, then at least $1/2$ of validators the prover claimed to sign $S$ did not. Therefore the proof fails with probability at least $2^{-k_{approval}}$.
+
+Furthermore, if signing an incorrect statement $S$ is slashable and we slash by at least $minsupport$, then if the light client reports the initial claim, then at least $minsupport$ stake can be slashed for an incorrect inital statement. Now if at least $2/3$ of validators are honest, then the proof fails with probability $2^{-k_{approval}}$ and so there is an expected cost of $2^{k_{approval}} minsupport$. 
+
+## Implementing this on a PoW Ethereum chain
```
Comment (vgeddes) [inline docs/markdown/concepts-and-architecture/polkadot-light-client/interactive-protocol.md:34]: Implementation on an Ethereum PoW Chain.

File: docs/markdown/building-with-snowbridge.md [hunk 31]
```diff
@@ -0,0 +1,31 @@
...
+has_children: true
+permalink: /building-with-snowbridge/
+---
+# Building With Snowbridge
+Snowbridge will support various bits of functionality for blockchain developers.
+
+## Running our Software
+If you're interested in running, using or building on any of our software directly, see our [GitHub](https://github.com/Snowfork/polkadot-ethereum){: target="_blank"}. We also have a guide to help with [Running and Testing the Bridge Locally](./running-locally)
+
+## Integrating with PolkaETH, PolkaERC20, or SnowDOT
```
Comment (vgeddes) [inline docs/markdown/building-with-snowbridge.md:14]: Integrating with our applications
